napari/layers/base/base.py

```diff
@@ -370,6 +377,7 @@ def __init__(
             status=Event,
             help=Event,
             interactive=Event,
+            mouse_pan_zoom_toggles=Event,
```

Comment:
```suggestion
            interactive=WarningEmitter(
                trans._("layer.events.interactive is deprecated since 0.5.0 and will be removed in 0.6.0. Please use layer.events.mouse_pan and layer.events.mouse_zoom", deferred=True), type_name='interactive')
            mouse_pan=Event,
            mouse_zoom=Event,
```